src/errors/utils/errorMapping.ts

```diff
@@ -0,0 +1,24 @@
+import { UserRejectionError } from '../userRejection';
+
+export function isUserRejectionError(error: Error): boolean {
+  const bearbyUserRejectionErrorMessage = 'User rejected';
+  const massaUserRejectionErrorMessage = 'Action canceled by user';
+  return (
+    error.message &&
+    (error.message.includes(bearbyUserRejectionErrorMessage) ||
+      error.message.includes(massaUserRejectionErrorMessage))
+  );
+}
+
+export function operationErrorMapping(
+  operationName: string,
+  error: Error,
+): Error {
+  if (isUserRejectionError(error)) {
+    return new UserRejectionError({
+      operationName: operationName,
```

Comment:
maybe you should keep the original error message. Here it is useless but it could be usefull for other errors containing infos